content/docs/components/ksonnet.md

```diff
@@ -44,6 +43,13 @@ Follow the steps below to install ksonnet:
       export PATH=$PATH:${HOME}/bin/$KS_PKG
       ```
 
+Here are the ksonnet package download links for each operating system:
+
+* [macOS](https://github.com/ksonnet/ksonnet/releases/download/v0.13.1/ks_0.13.1_darwin_amd64.tar.gz)
```

Comment:
Although you introduced the list of links rather clearly, it's easy to skim over the fact that these are download links rather than links to another doc page with OS-specific instructions.

For increased clarity, consider using text like "Download ksonnet for macOS", and so on, for each OS-specific link. It sounds a little redundant, but it gives a clearer warning (on each link) that you're about to download a package file.

Reply:
Done.